app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -160,7 +161,7 @@ module.exports = React.createClass
           disabled = @getDisabledAttribute(i)
           selectedOption = if @state.selectedOptions[i]?.value then @state.selectedOptions[i] else null
           optionsKeys.push selectedOption.value if selectedOption?.value
-          <div key={select.id}>
+          <div id={select.id} key={select.id}>
```

Comment:
=/ Still can't get to work, not sure what I'm doing wrong. I'm sure something stupid right in front of me.